diff --git a/node_modules/@twotalltotems/react-native-otp-input/dist/index.js b/node_modules/@twotalltotems/react-native-otp-input/dist/index.js
index 71580b8..55be344 100644
--- a/node_modules/@twotalltotems/react-native-otp-input/dist/index.js
+++ b/node_modules/@twotalltotems/react-native-otp-input/dist/index.js
@@ -104,14 +104,16 @@ export default class OTPInputView extends Component {
                 }
             }
         };
-        this.focusField = (index) => {
-            if (index < this.fields.length) {
-                this.fields[index].focus();
-                this.setState({
-                    selectedIndex: index
-                });
-            }
-        };
+        this.focusField = index => {
+      if (index < this.fields.length) {
+        setTimeout(() => {
+          this.fields[index].focus();
+        });
+        this.setState({
+          selectedIndex: index,
+        });
+      }
+    };
         this.blurAllFields = () => {
             this.fields.forEach((field) => field.blur());
             this.setState({
